test/specs/parser/spec.js

```diff
@@ -0,0 +1,82 @@
+var includes = require('./../../utils/ermrest-init.js').init();
```

Comment:
Hey, what you've done is correct, but instead of writing your testcases in the `spec.js` file, you need to create a `tests` folder inside your `parser` folder, which will contain the `01.parser.js` test file.

For reference, you can check the updated wiki on testing [here](https://github.com/informatics-isi-edu/ermrestjs/wiki/Unit-Testing-In-ermrestjs). It mentions the file structure.

After making changes according to the wiki, your `spec.js` file will look like this

```
require('./../../utils/starter.spec.js').runTests({
    description: 'In sample spec, ',
    testCases: [
        "/sample/tests/01.parser.js",
    ],
    schemaConfigurations: []
});
```

For your scenario, you're not depending on a schema, so you don't need to create the `conf` folder and add the `parse.conf.json` file.